README.md

```diff
@@ -244,6 +244,40 @@ type Output struct {
 }
 ```
```

Comment:
Another test case for verifying that the error path is correctly built:
```go
package structs

// goverter:converter
type Converter interface {
    // goverter:fold
    Convert(source Input) Output
}

type Input struct {
    NESTED InputNested
}
type InputNested struct {
    Name string
}
type Output struct {
    Nested OutputNested
}
type OutputNested struct {
    Name int
}
```
The error path should look like:
```
| github.com/jmattheis/goverter/execution.Input
|
|      | github.com/jmattheis/goverter/execution.InputNested
|      |
|      |      | string
|      |      |
source.NESTED.Name
target.Nested.Name
|      |      |
|      |      | int
|      |
|      | github.com/jmattheis/goverter/execution.OutputNested
|
| github.com/jmattheis/goverter/execution.Output
```

Reply:
👍